docs/index.html

```diff
@@ -129,7 +143,19 @@ <h1>Himalaya</h1>
             </label>
           </div>
           <div class='pane pane-output'>
-            <pre id='output'></pre>
+            <pre id='output-parse'></pre>
+          </div>
+        </div>
+        <div class='split' id="stringify-pane" style="display: none;">
+          <div class='pane-header'>
+            <label>Stringify</label>
+            <label>
+              <input id='prefer-double-quote-attributes' type='checkbox' />
+              <span>Prefer Double Quote Attributes</span>
```

Comment:
All other checkboxes are sentence case which is the better format
```suggestion
              <span>Prefer double quote attributes</span>
```